static/css/styles.css

```diff
@@ -0,0 +1,13 @@
+.row {
+    margin-top: 20px;
+}
+.sidebar {
+    list-style: none;
+    padding: 0;
+}
+h3 {
+    padding-bottom: 5px;
+}
+a {
+    color: navy;
+}
```

Comment:
add blank lines to the end of the files where you don't have it

Reply:
Done!